README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a project for microverse 2.0 training
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://test.com)
```

Comment:
- [ ] Kindly provide a valid Live Demo Link for your project or just delete it from the README file since it's not required for this project to have a Live Demo Link.
This can mislead future viewers!